common/lib/core-interfaces/src/handles.ts

```diff
@@ -52,7 +53,7 @@ export interface IProvideFluidHandle {
  */
 export interface IFluidHandle<
     // REVIEW: Constrain `T` to `IFluidObject & IFluidLoadable`?
```

Comment:
I bet you could remove this comment and constrain `T extends FluidObject` now, if that is even worth doing as a hint (or just delete the comment)

Reply:
yeah. handles are weird. i don't even really know what the right typing should be. we also use if for DDS which don't use the fluid object pattern, but maybe they should? not sure